### DISCUSSION TITLE
Unconditional Basic Income (UBI) for Nation3 Citizens

### DISCUSSION THREAD

**Discussion by aahna-ashina** [nation3/n3bi#1] [Ideas]
@yukiwaki Let's continue the [Discord](https://discord.com/channels/690584551239581708/963778350814478346/992088972165644358) discussion here 😄

**Comment by afurmanczyk** [comment]
Love the topic of a UBI so thanks for starting this @aahna-ashina and @yuki. 

I first started reading books on UBI in 2015 and the conclusion I came to was that UBI probably isn't viable without an automated robot work force creating the physical goods, and materials needed to support that society, unless that UBI was only for a small select group of people. 

However, if you need to exclude people from a UBI then that would be taking the U (universal) out of UBI. If many people work to supply goods and value for a few, then that sounds more like an exploitative setup.

In order to create a sustainable UBI, that value must come from the product of somewhere/something. 

There was talk about the money could coming from VC investing, but wouldn't that make N3 totally dependant on trade with other producers to buy our own goods to live off of? I wonder if a UBI should be surplus from automated labor? In other words, our UBI's are generated from the bounty of a robot/artificial work force working for N3.

**Reply by aahna-ashina** [reply]
@afurmanczyk Good point, it's not really _universal_ if it's only for 420 Nation3 citizens.  So maybe we can name it something else?  How about _Nation3 Basic Income_ (`N3BI`/`NBI`)?

**Comment by yukiwaki** [comment]
Hi! @afurmanczyk @aahna-ashina 

I agree with "NBI" since it's limited to 420 Nation3 citizens.

@afurmanczyk 
>If many people work to supply goods and value for a few, then that sounds more like an exploitative setup.
What do you mean by this?

I think it's nice to start with a small group of people as an experiment but we may need to define the purpose of NBI or this experiment first.
For me, it's like we can unleash creativity by having a minimum but stable income without any condition, and not working just for money. 
But what about yours?

**Comment by luisivan** [comment]
Having UBI seems indeed like the endgoal, a few caveats are:
- UBI indeed works when there's an automated, non-human workforce creating value, as @afurmanczyk mentioned. Otherwise it's just freeriding. Nation3 is still not at the stage where the nation has a prosperous internal economy let alone automated ways of creating value.
- The closest we have to an automated way of making money is crypto protocols. Once they enforce a series of constraints and achieve usage, the incentive scheme runs on its own and accrues value.

Building upon the previous two points, the idea to provide some UBI (or `N3BI`) to Genesis Citizens, who are committing something (bearing cost of capital) makes sense, since they are clearly not freeriders. One thing to consider here is the endgoal with `N3BI`. UBI's goal is to provide some basic financial freedom, generally so that people can be more relaxed, enjoy life more, and even be more creative (which might result in economic upside for the nation). Here's different because the amount will be small in the beginning. One possible outcome is that Genesis Citizens might re-stake that $NATION or use it inside the internal economy (e.g. to make agreements inside the upcoming Court). In that case, I would hold off on `N3BI` until we have some ways that people could use those tokens inside Nation3's internal economy. Thoughts?

**Reply by yukiwaki** [reply]
Yeah I agree with you, it's like the endgoal and better to start N3BI when we have ways people use tokens inside Nation3's internal economy.

I'm just curious but what kind of internal usages are you thinking so far?

**Reply by luisivan** [reply]
The [Court system](https://forum.nation3.org/t/nation3-court-a-backbone-to-bootstrap-a-circular-economy-by-increasing-trust-between-citizens/553) could be a great way to incentivize citizens to transact with each other, generating a circular economy

**Reply by afurmanczyk** [reply]
I agree, I think it's not viable in the short term, but an N3BI can be a longer term goal.
